src/tr/lilyumfansub/src/eu/kanade/tachiyomi/extension/tr/lilyumfansub/LilyumFansub.kt

```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.extension.tr.lilyumfansub
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class LilyumFansub : Madara("LilyumFansub", "https://lilyumfansub.com.tr", "tr", SimpleDateFormat("d MMMMM yyyy", Locale("tr")))
```

Comment:
the website appear to use the `newChapterEndpoint`.
Also this date format seems to be wrong, considering it looks like month day, year on the site?